src/js/components/banner.js

```diff
@@ -1,21 +1,23 @@
-var select = require('../utils/select');
-var dispatch = require('../utils/dispatch');
+'use strict';
+const behavior = require('../utils/behavior');
+const toggle = require('../utils/toggle');
 
-function bannerClickHandler (event) {
-  (event.preventDefault) ? event.preventDefault() : event.returnValue = false;
+const PREFIX = require('../config').prefix;
+const HEADER = `.${PREFIX}-banner-header`;
+const EXPANDED_CLASS = `${PREFIX}-banner-header-expanded`;
 
-  this.classList.toggle('usa-banner-header-expanded');
-}
+const toggleBanner = function (event) {
+  event.preventDefault();
+  this.closest(HEADER).classList.toggle(EXPANDED_CLASS);
```

Comment:
Is `closest` polyfilled? I think it needs to be for IE. http://caniuse.com/element-closest